app/assets/stylesheets/alchemy/icons.scss

```diff
@@ -11,7 +11,7 @@
     font-size: 19px;
   }
 
-  &.add_page {
+  &.add_page, &.add_node {
```

Comment:
Selector `add_page` should be written in lowercase with hyphens<br>Selector `add_node` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own single line